src/daniil_efremov_ex/ex01/tidy_code.py

```diff
@@ -0,0 +1,39 @@
+__author__ = 'Daniil Efremov'
+__email__ = 'daniil.vitalevich.efremov@nmbu.no'
+
+from random import randint as rand_num
+
+
+def negative_checker():
+    guessed_int = 0
+    while guessed_int < 1:
+        guessed_int = int(input('Your guess: '))
+    return guessed_int
+
+
+def rand_int():
+    return rand_num(2, 12)
```

Comment:
Nice thinking, but can you spot the differences between `randint(2,12)` and `randint(1,6) + randint(1,6)`?
In `randint(2,12)`, the chances to get 2 or 12 value is 1/11, while in `randint(1,6) + randint(1,6)` that chance is 1/36